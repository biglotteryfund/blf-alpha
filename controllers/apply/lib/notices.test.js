/* eslint-env jest */
'use strict';
const {getNoticesSingle } = require('./notices');


test.each(['school', 'college-or-university', 'statutory-body'])(
    'get notices for under Â£10,000 application in England for %p',
    function (orgType) {
        const resultSingle = getNoticesSingle('en', {
            formId: 'awards-for-all',
            applicationData: {
                projectCountry: 'england',
                organisationType: orgType,
            },
        });
        expect(resultSingle).toMatchSnapshot();
        expect(resultSingle).toHaveLength(1);
    }
);
