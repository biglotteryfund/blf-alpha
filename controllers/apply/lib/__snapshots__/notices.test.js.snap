// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`get notices for under £10,000 application in England for "college-or-university" 1`] = `
Array [
  Object {
    "body": "We've changed our eligibility criteria (for the time being) to help communities through the pandemic. So for funding under £10,000, we're only funding voluntary and community organisations with COVID-19 related projects.",
    "title": "We're sorry, but your application is now not eligible for funding",
  },
]
`;

exports[`get notices for under £10,000 application in England for "school" 1`] = `
Array [
  Object {
    "body": "We've changed our eligibility criteria (for the time being) to help communities through the pandemic. So for funding under £10,000, we're only funding voluntary and community organisations with COVID-19 related projects.",
    "title": "We're sorry, but your application is now not eligible for funding",
  },
]
`;

exports[`get notices for under £10,000 application in England for "statutory-body" 1`] = `
Array [
  Object {
    "body": "We've changed our eligibility criteria (for the time being) to help communities through the pandemic. So for funding under £10,000, we're only funding voluntary and community organisations with COVID-19 related projects.",
    "title": "We're sorry, but your application is now not eligible for funding",
  },
]
`;
